### PR TITLE
Release prep for 4.4.0

### PR DESCRIPTION
**Pull request by willmeek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](http://semver.org).
 ## Supported Release [4.3.0]
```
Comment (pmcmaw) [inline CHANGELOG.md]: This will need updated.

Comment (pmcmaw) [inline CHANGELOG.md]: Needs updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](http://semver.org).
...
+This release is a rollup of new features and fixes.
+
+#### Added
+- Install `apt-transport-https` if using Debian 8, 9 or Ubuntu 14.04, 16.04.
+- Adds a boolean option `direct` to proxy settings to bypass `https_proxy` if not set.
+- Adds facter facts for `dist-upgrade` apt updates.
+
+#### Changed
+- Update class is now private.
+- (MODULES-4265) Detect security updates from multiple sources.
```
Comment (pmcmaw) [inline CHANGELOG.md]: This can include a hyperlink to the ticket.

File: CHANGELOG.md (hunk 1 above)
Comment (willmeek) [reply CHANGELOG.md]: D'oh! How did I miss that! Good catch!

File: CHANGELOG.md (hunk 2 above)
Comment (willmeek) [reply CHANGELOG.md]: Added as hyperlink. Moved to 'Fixed' section as it looks more like a bug fix.

Comment (pmcmaw) [reply CHANGELOG.md]: I don't think you want the stars in front of the hyperlink.

File: CHANGELOG.md (hunk 1 above)
Comment (pmcmaw) [reply CHANGELOG.md]: I would also include the link at the bottom of the file eg:

[4.4.0]:https://github.com/puppetlabs/puppetlabs-apt/compare/4.3.0...4.4.0

File: CHANGELOG.md (hunk 2 above)
Comment (willmeek) [reply CHANGELOG.md]: Removed the asterisks, was copy and pasting the format from an earlier build note but it didn't really work.

File: CHANGELOG.md (hunk 1 above)
Comment (willmeek) [reply CHANGELOG.md]: Good call, I'll keep an eye to do that in future

**Pull request merged**